test/cpp/torch_xla_test.cpp

```diff
@@ -51,13 +47,43 @@ void XlaTest::ExpectCounterNotChanged(
   EXPECT_TRUE(changed.empty());
 }
 
+void XlaTest::ExpectCounterNotChanged(
+    const std::string& counter_regex,
+    const std::unordered_set<std::string>* ignore_set) {
+  MakeEndSnapshot();
+  auto changed =
+      start_msnap_->CounterChanged(counter_regex, *end_msnap_, ignore_set);
+
+  ExpectCounterNotChanged_(changed);
+
+  // Some operators could've been renamed to `opName_symint`, yet the tests are
+  // using the old names. We modify `ExpectCounterNotChanged` to also check
+  // `opName_symint` counters. When we finish migrating the ops to symints, we
+  // would remove this logic and fix all the tests
+  auto changed_symint =
+      start_msnap_->CounterChanged(counter_regex, *end_msnap_, ignore_set);
```

Comment:
Why is this line not saying `counter_regex + "_symint"`? I am trying to understand the diff between line `54` and `63` here.

CC @JackCaoG @Krovatkin

Reply:
I will push a PR shortly.